assets/css/components/_page-header.scss

```diff
@@ -0,0 +1,10 @@
+.page-header {
+  display: flex;
+  justify-content: space-between;
+  height: rem(50px);
+  margin-bottom: rem(15px);
+
+  &__title {
+    font-size: map-get($font-size, 'xxxl');
```

Comment:
I saw only `sm`, `md` and `xl` in the variables file. Is this supposed to be `xl`? 🤔

Reply:
@rosle fixed in [2a5b032](https://github.com/rafayet-monon/elixir_search_extractor/pull/36/commits/2a5b032c7a09dbbe2054cf271cfb587b79144810)